content/5-shape-elements/default.txt

```diff
@@ -1412,7 +1412,7 @@ marker.width(10)
 <br>
 # SVG.Style
 
-The Style-element creates a style-tag and atts rules for a selector
```

Comment:
Funny that you didnt catch the `atts` typo here :D. I guess that should be `adds` ^^

Reply:
Oh wow, right in front of my eyes and still missed it haha! 🙈 

All fixed :)